joomla/joomla-cms
# [com_fields] Fix for cropped 'Show User Custom Field'

Pull Request for Issue #13594 .

### Summary of Changes
Stop 'Show User Custom Field' value from been cropped

### Testing Instructions
Apply patch and navigate to #Contacts: Edit' -> Tab 'Display' -> Field "Show User Custom Field" Text.

Check input value is not cropped (see #13594)
